bin/mqtt_bridge_run.sh

```diff
@@ -10,4 +10,4 @@ then
       MQTT_BRIDGE_LOG4J_OPTS="-Dlog4j2.configurationFile=file:${MYPATH}/../config/log4j2.properties"
 fi
 
-exec java $MQTT_BRIDGE_LOG4J_OPTS -classpath "${MYPATH}/../libs/*" io.strimzi.kafka.bridge.mqtt.Main "$@"
+exec java "$MQTT_BRIDGE_LOG4J_OPTS" -classpath "${MYPATH}/../libs/*" io.strimzi.kafka.bridge.mqtt.Main "$@"
```

Comment:
why did you need to add double quotes here? Compared to the Strimzi HTTP bridge we didn't need them.
You are also missing to pass the `$JAVA_OPTS` env var at the beginning. It's not filled in this script but it could be filled by the docker related script calling this one.

Reply:
Does not double quotes prevent globbing and word splitting?

Reply:
but isn't it defined as a string already with double quotes?

Reply:
Yeah, but I keep getting warnings regarding it.

Reply:
which warnings? from where?

Reply:
@ppatierno regarding the $JAVA_OPTS, was it not related to STRIMZI_ ? Because its first value was coming from `get_gc_opts` and inside this function we used `STRIMZI_GC_LOG_ENABLED`. 
Please clarify, maybe I got it wrong.

Reply:
![image](https://github.com/strimzi/strimzi-mqtt-bridge/assets/42675180/cb085da5-b205-4a52-b2d8-29ddeadad9de)
the warnings

Reply:
> @ppatierno regarding the $JAVA_OPTS, was it not related to STRIMZI_ ? Because its first value was coming from get_gc_opts and inside this function we used STRIMZI_GC_LOG_ENABLED.
Please clarify, maybe I got it wrong.

You are actually setting it in the bin/docker script for Max heap and FIPS but not using here. So you need it to be added on this command line :-)

> the warnings
I see but in the Strimzi HTTP bridge we had to disable this warning at some point (raised by shellcheck) because of some TLS related commands with kytool. Because we'll have TLS support one day in the future, I would follow the same pattern here.